app/routes.js

```diff
@@ -37,6 +37,9 @@ require('./routes/survey')(router)
 // Clear all data in session if you open /admin/clear-data
 router.post('/admin/clear-data', function (req, res) {
   req.session.data = {}
+
+  console.log('hello')
```

Comment:
👋

Reply:
Oops. Was testing why the clear data page no longer works. It’s got me flummoxed!